src/assets/files/data.json

```diff
@@ -22,5 +22,11 @@
         "name": "cli algorithms visualizer",
         "description": "Simple CLI-based Algorithms Visualizer",
         "url": "https://github.com/go-nerds/cli-algorithms-visualizer"
+    },
+    {
+        "id": 5,
+        "name": "rapid-express",
```

Comment:
Thank you for your PR!
Can you please add the name without any symbols e.g. `rapid express`?